beacon/goclient/aggregator.go

```diff
@@ -18,15 +17,6 @@ func (gc *GoClient) SubmitAggregateSelectionProof(slot phase0.Slot, committeeInd
 	// https://github.com/ethereum/consensus-specs/blob/v0.9.3/specs/validator/0_beacon-chain-validator.md#broadcast-aggregate
 	gc.waitToSlotTwoThirds(slot)
 
-	// differ from spec because we need to subscribe to subnet
-	isAggregator, err := isAggregator(committeeLength, slotSig)
-	if err != nil {
-		return nil, DataVersionNil, fmt.Errorf("failed to check if validator is an aggregator: %w", err)
-	}
-	if !isAggregator {
-		return nil, DataVersionNil, fmt.Errorf("validator is not an aggregator")
-	}
```

Comment:
Doing this here seems to be causing this error consistently showing in logs:
```
{
  "level": "\u001b[35mDEBUG\u001b[0m",
  "time": "2024-10-25T13:17:08.000420Z",
  "name": "Controller.Validator",
  "msg": "❗ could not handle message",
  "pubkey": "9203935ee396795ab6775b910f94d8d38850cd2331fc4416fdfeb87cc6d4cd764a9f6b7dcddd05b6a9c76a73cd68f100",
  "beacon_role": "AGGREGATOR",
  "signer": 419,
  "slot": 2829985,
  "msg_type": "partial_signature",
  "error": "failed to submit aggregate and proof: validator is not an aggregator",
  "errorVerbose": "validator is not an aggregator\nfailed to submit aggregate and proof\ngithub.com/ssvlabs/ssv/protocol/genesis/ssv/runner.(*AggregatorRunner).ProcessPreConsensus\n\t/go/src/github.com/ssvlabs/ssv/protocol/genesis/ssv/runner/aggregator.go:104\ngithub.com/ssvlabs/ssv/protocol/genesis/ssv/validator.(*Validator).ProcessMessage\n\t/go/src/github.com/ssvlabs/ssv/protocol/genesis/ssv/validator/validator.go:151\ngithub.com/ssvlabs/ssv/protocol/genesis/ssv/validator.(*Validator).ConsumeQueue\n\t/go/src/github.com/ssvlabs/ssv/protocol/genesis/ssv/validator/msgqueue_consumer.go:154\ngithub.com/ssvlabs/ssv/protocol/genesis/ssv/validator.(*Validator).StartQueueConsumer\n\t/go/src/github.com/ssvlabs/ssv/protocol/genesis/ssv/validator/msgqueue_consumer.go:60\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1695"
}
```
so,
- moving this code out of `goclient` because it doesn't really belong here
- and handling `isAggregator==true` a bit differently (just `log.Debug` instead of erorring)